api/tools/src/main/java/org/pmiops/workbench/tools/ExportWorkspaceData.java

```diff
@@ -269,6 +297,39 @@ private WorkspaceExportRow toWorkspaceExportRow(DbUser user) {
             .map(dateFormat::format)
             .orElse(""));
     row.setCreatorRegistrationState(user.getDataAccessLevelEnum().toString());
+    row.setDegrees(
+        Optional.ofNullable(user.getDegreesEnum()).orElse(ImmutableList.of()).stream()
+            .map(Degree::toString)
+            .collect(Collectors.joining(",")));
+    DbDemographicSurvey demo = user.getDemographicSurvey();
```

Comment:
nit: i think you can just move this object into the conditional and drop the null check

Reply:
`includeDemographics` is driven by a flag - not null'ness of the demographics value. So this doesn't work - unless I'm misunderstanding the suggestion.